chanzuckerberg/single-cell-data-portal
# chore: Make single python requirements file for each dockerfile

## Reason for Change

- #5024 
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

- add appropriately named requirements file for each dockerfile at the root of repository
- modify the docker files to **only** include these new requirements file and install them
- remove deprecated `requirements.txt` files
- update README documentation to reflect changes

## Testing steps

- Run `make local-sync` and check that all docker images are reconstructed
- `pip install -r <requirement-files>` in a _local python virtual environment_ and unit tests
- Let CI pipeline on GHA reconstruct the docker images and run the pipeline successfully

## Notes for Reviewer
